.github/workflows/build.yml

```diff
@@ -19,33 +16,28 @@ jobs:
     name: Build on ${{matrix.os}}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Install Java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
-          java-version: 11
+          distribution: 'zulu'
+          java-version: '11'
       - name: Install dependencies
         if: matrix.os == 'ubuntu-latest'
         run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libgtk-3-dev nodejs chromium-browser
-      - name: Set up cache
```

Comment:
Why did you remove this?

Reply:
`gradle/gradle-build-action` manages caches keys by itself, better to rely on Gradle team rather than managing it ourselves.

Reply:
I thought that GitHub Actions drops everything including cache folders by default. But if you believe it's ok then lgtm.

Reply:
[gradle/gradle-build-action](https://github.com/gradle/gradle-build-action#caching)

Reply:
TIL